### ISSUE TITLE
macOS menubar Window option position & localization

### ISSUE THREAD

**Issue by Jerry23011** [moneymanagerex/moneymanagerex#5344]
MMEX version:
 - [x] 1.6.1

Operating System:
 - [ ] Windows
 - [x] Mac OSX
 - [ ] Linux 

# Description of the bug
The Window option under macOS is not localizable, and it seems that it is not in the right position

**Simplified Chinese** interface
<img width="857" alt="截屏2022-11-18 18 48 31" src="https://user-images.githubusercontent.com/89069957/202693621-84cb2b74-4c40-46c3-bbe7-dd8eced75bd5.png">
**English** interface
<img width="657" alt="截屏2022-11-18 19 14 29" src="https://user-images.githubusercontent.com/89069957/202693648-7e803d2c-35f7-42a4-9a48-31cbc2e722df.png">

Below is how other macOS apps usually place the Window option
<img width="576" alt="截屏2022-11-18 19 14 39" src="https://user-images.githubusercontent.com/89069957/202693796-1d8e03d3-57d7-4647-b71b-4096a43825eb.png">

# Reproduction
Switch the language of mmex to other than English (eg. Simplified Chinese)
Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Expected result:
Window option before Help in the menubar and with localizable string

Actual result:
Position of Window option is at last and not localizable

**Comment by whalley** [comment]
Already being discussed here: https://github.com/wxWidgets/wxWidgets/issues/22870

Will have to wait for wx3.2.2, I'll put this on hold for now.

**Comment by Jerry23011** [comment]
Hi, thanks a lot, looking forward to it.
